scripts/automatic_tag.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/transport/ssh"
+	"golang.org/x/mod/semver"
+)
+
+const (
+	sdkRepo      = "git@github.com:stackitcloud/stackit-sdk-go.git"
+	patch        = "patch"
+	minor        = "minor"
+	coreOnlyFlag = "--core-only"
+)
+
+var (
+	updateTypes = []string{minor, patch}
+	usage       = fmt.Sprintf("go run automatic_tag.go [minor|patch] ssh-private-key-file-path password %s", coreOnlyFlag)
```

Comment:
I think that doesn't make a lot of sense because the script can also be run directly and it seems strange that it would give the usage of the make command

Wrong usage of the make command is not part of the scope of the script